src/app/core/models/issue.model.ts

```diff
@@ -105,7 +110,7 @@ export class Issue {
 
     issue.githubComments = githubComments;
     issue.issueComment = template.comment;
-    issue.teamResponse = template.teamResponse !== undefined ? template.teamResponse.content : undefined;
+    issue.teamResponse = Issue.updateTeamResponse(template.teamResponse.content);
```

Comment:
``I have added default value to empty Github responses which should fix the issue.``